src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,22 @@
+@mixin flex-center() {
+  display: flex;
+  justify-content: space-between;
```

Comment:
The `flex-center` mixin uses `justify-content: space-between;`, which might not always center the content as expected. If the intention is to center items, consider using `justify-content: center;` instead. Otherwise, ensure that the mixin is used in contexts where space-between is desired.